packages/tests/silverback-gatsby/playwright-tests/references.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  drupal,
+  drupalLogin,
+  gatsby,
+  resetState,
+  waitForGatsby,
+} from '@amazeelabs/silverback-playwright';
+import { expect, test } from '@playwright/test';
+
+test.beforeAll(async () => {
+  await resetState();
+});
+
+test('@gatsby-build references', async ({ page }) => {
```

Comment:
It should fallen on https://github.com/AmazeeLabs/silverback-mono/pull/1431/commits/83048e97bac95042b3f03430facc51c8551d6ae5 yet it dod not. I don't know why.

Locally it fails properly. But in CI an incremental Gatsby build sources all nodes for some reason.

Another reason to get rid of `@amazeelabs/silverback-playwright` ASAP. Will do in a separate PR.